pkg/users/index.html

```diff
@@ -128,13 +128,13 @@ <h1 translatable="yes">Account not available or cannot be edited.</h1>
                  data-target="#password-expiration"></a>
             </td>
           </tr>
-          <tr id="session_recoding_tr">
+          <tr id="session_recoding_tr" style="display:none;">
```

Comment:
This smells like inline style and thus violating `no-inline` content-security-policy? But indeed the pkg/playground uses this, so maybe that is a special case. Anyway, `hidden` is much more standard and also shorter.